compiler/scenario-service/server/src/main/scala/com/digitalasset/daml/lf/ScenarioServiceMain.scala

```diff
@@ -72,6 +72,7 @@ object ScenarioServiceMain extends App {
 
 object ScenarioService {
   private def notFoundContextError(id: Long): StatusRuntimeException =
+  // TODO error codes: Adapt V2 ?
```

Comment:
Language team

Reply:
Could you be a bit more explicit about what you expect form Daml connect teams here ? 

Do you want to know if those error could happen ? If there is some missing ?

Reply:
scenarios will die completely, safe to ignore.